src/rr_trace.capnp

```diff
@@ -110,6 +110,10 @@ struct Header {
     # Whether the version of rr that recorded this saved the extra registers
     # for the pkey_alloc syscall.
     pkeyAllocRecordedExtraRegs @20 :Bool = false;
+
+    # Whether the version of rr that recorded this forced a tick after each
+    # buffered syscall.
+    bufferedSyscallForcedTick @21 :Bool = false;
```

Comment:
I like it better if the `quirk` is true in the old version and false in the new version, such that a trace produced by a current version of rr has no "quirks", but that's a minor preference.

Reply:
I actually agree with you but roc wanted it the other way when I did the pkey_alloc(2) quirk above so that there wasn't a negative in the quirk (e.g. "bufferedSyscallDidntForceTick").